src/pretix/presale/templates/pretixpresale/event/base.html

```diff
@@ -57,6 +60,18 @@
 {% endblock %}
 {% block page %}
     <div class="page-header {% if event_logo and event.settings.logo_image_large %}logo-large{% endif %}">
+        {% if event.settings.locales|length > 1 %}
+            {% if event.settings.theme_color_background|upper == "#FFFFFF" and not event.settings.logo_image_large %}
+                <div class="{% if not event_logo or not event.settings.logo_image_large %}pull-right flip{% endif %} loginbox hidden-print">
+                    <nav class="locales" aria-label="{% trans "select language" %}">
+                        {% for l in languages %}
+                            <a href="{% url "presale:locale.set" %}?locale={{ l.code }}&next={{ request.path }}{% if request.META.QUERY_STRING %}%3F{{ request.META.QUERY_STRING|urlencode }}{% endif %}" class="{% if l.code == request.LANGUAGE_CODE %}active{% endif %}" rel="nofollow" lang="{{ l.code }}" hreflang="{{ l.code }}">
+                                {{ l.name_local }}</a>
+                        {% endfor %}
+                    </nav>
+                </div>
+            {% endif %}
+        {% endif %}
```

Comment:
Any reason you moved this?

Generally, I have no problem with moving it above the logo consistently, but that's going to need more work. At least one configuration looks weirder now.

Before:

![2021-02-19-160042_904x187_scrot](https://user-images.githubusercontent.com/64280/108521366-f2101580-72cb-11eb-88b6-9f828832d254.png)

After:

![2021-02-19-160007_821x269_scrot](https://user-images.githubusercontent.com/64280/108521370-f2a8ac00-72cb-11eb-9935-767b3d65d22d.png)

It's no longer visible which language is active and the languages and link back being on the same height looks a little off. If there's no strong reason to move it, we should maybe tackle this separately. (I do see potential for simplification here.)

Reply:
Sorry, that might be an artifact of me trying to get everything header-related into the <header>-Element and have a „better“ source-order. I think, we should revert.

Reply:
:+1: